template.distillery/Makefile.os

```diff
@@ -46,7 +46,8 @@ endif
 .PHONY: all css byte opt
 
 DIST_DIRS          := $(ETCDIR) $(DATADIR) $(LIBDIR) $(LOGDIR) \
-                      $(FILESDIR)/avatars/tmp $(ELIOMSTATICDIR)
+                      $(FILESDIR)/avatars/tmp $(ELIOMSTATICDIR) \
+                      $(shell dirname $(CMDPIPE)) $(ELIOMTMPDIR)
```

Comment:
Where is `ELIOMTMPDIR` defined?

Reply:
Indeed, ELIOMTMPDIR seems to be set to /tmp in config file without any variable ...
It is probably not needed. I am still not completely sure for static dir but at least it compiles now.